mu-plugins/blocks/query-filter/src/view.js

```diff
@@ -1,8 +1,7 @@
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
```

Comment:
The module build is not compatible with importing from other `@wordpress/*` packages, trying was giving me this error:

```
Module not found: Error: Attempted to use WordPress script in a module: @wordpress/i18n, which is not supported yet.
```

The workaround is to pass the label as a data attribute on the element from `render.php`. I don't think we'll need to do this forever, as this is currently being iterated on.